src/transport/SessionHandle.cpp

```diff
@@ -23,12 +23,59 @@ namespace chip {
 
 using namespace Transport;
 
+using AuthMode          = Access::AuthMode;
 using SubjectDescriptor = Access::SubjectDescriptor;
 
 SubjectDescriptor SessionHandle::GetSubjectDescriptor() const
 {
-    SubjectDescriptor subjectDescriptor = { .fabricIndex = mFabric };
-    // TODO: fill subject descriptor with proper fields
+    SubjectDescriptor subjectDescriptor;
+    if (IsSecure())
+    {
+        if (IsOperationalNodeId(mPeerNodeId))
+        {
+            // Currently ending up here after commissioning (good)
+            subjectDescriptor.authMode    = AuthMode::kCase;
+            subjectDescriptor.subject     = mPeerNodeId;
+            subjectDescriptor.fabricIndex = mFabric; // have fabric here
+            // TODO: add CATs
```

Comment:
Add `// TODO(#issue):` for follow-up and tracking purposes.

Reply:
There is already an issue for CATs in SubjectDescriptor, I added it to the TODO.